static-website/static-website.yaml

```diff
@@ -35,6 +35,7 @@ Metadata:
       Parameters:
       - DefaultRootObject
       - DefaultErrorPagePath
+      - DefaultErrorResponseCode
```

Comment:
Is `NotFoundErrorResponseCode` a better name for the parameter?

Reply:
It might be. but it is inconsistent with DefaultErrorPagePath. So I rather would keep the new parameter name consistent.